css/style.css

```diff
@@ -1071,3 +1356,1390 @@ ul {
 ::placeholder {
   color: #b3bac5;
 }
+
+/* desktop version */
+@media (min-width: 1000px) and (orientation: landscape) {
+  /* For desktop: */
+  #container {
+    width: 100%;
+    aspect-ratio: 1440 / 5125;
+    position: absolute;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-size: 100%;
+    margin: 0;
+    box-sizing: border-box;
+    overflow-x: hidden;
+    background: #e5e5e5;
+    background: transparent;
+    background-image: url("images/desktop.jpg");
+    background-size: cover;
+    background-position: center;
+  }
+
+  #app-bar-compact {
+    width: 100%;
+    aspect-ratio: 1440 / 72;
+    z-index: 3;
+  }
+
+  #div-logo {
+    width: 6.944444%;
+    aspect-ratio: 100 / 40;
+    left: 9.930556%;
+    top: 0.292683%;
+    position: absolute;
```

Comment:
Kindly modify the logo to not have an underline just as it appears in the design template.

Your design:
![image](https://user-images.githubusercontent.com/20554583/194478596-d99b2123-c786-49f9-a5c9-89a8d44d01c9.png)

Figma template:
![image](https://user-images.githubusercontent.com/20554583/194478622-ec3220e3-ff21-4836-8f6a-a0bdb0bc5ca3.png)